packages/synapse-react-client/package.json

```diff
@@ -33,12 +33,12 @@
     "@brainhubeu/react-carousel": "1.19.26",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
-    "@mui/icons-material": "^5.14.16",
-    "@mui/material": "^5.14.17",
-    "@mui/system": "^5.14.17",
-    "@mui/utils": "^5.14.17",
-    "@mui/x-data-grid": "5.17.26",
-    "@mui/x-date-pickers": "^6.18.1",
+    "@mui/icons-material": "^5.15.10",
+    "@mui/material": "^5.15.10",
+    "@mui/system": "^5.15.9",
+    "@mui/utils": "^5.15.9",
+    "@mui/x-data-grid": "^6.19.4",
```

Comment:
Upgrade @mui/x-data-grid to v6. Like x-date-pickers below, it's ok to mix v6 with @mui/material v5

This change only affects challenge portal components.